beater/config.go

```diff
@@ -116,17 +117,26 @@ func (s *SourceMapping) isSetup() bool {
 	return s != nil && (s.EsConfig != nil)
 }
 
-func (c *FrontendConfig) memoizedSmapMapper() (sourcemap.Mapper, error) {
-	smap := c.SourceMapping
-	if !c.isEnabled() || !smap.isSetup() {
+func (c *Config) SetRumConfig() {
+	if c.RumConfig != nil && c.RumConfig.Enabled != nil {
+		return
+	}
+	if c.FrontendConfig != nil && c.FrontendConfig.Enabled != nil {
```

Comment:
L124 is unnecessary (as https://github.com/elastic/apm-server/pull/1155/files#diff-5142cb122ddb0cf46ec3a72f64f52591R197)